### PR TITLE
[CN-209] Merge 5.0 based changes:

### PR DESCRIPTION
**Pull request by hasancelik**
- all enhancements from 5.0-snapshot branch like multiple tag support and version updates
- modify existing github actions with multi tag support

### CODE REVIEW

**Review by SeriyBg — CHANGES_REQUESTED** [review]
One small comment, other than that LGTM

File: .github/workflows/4-1-z-backport_vulnerability_scan copy.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: Scheduled 4.1.z Vulnerability Scan
```
[the remark this answers is not in the document]
Comment (hasancelik) [reply .github/workflows/4-1-z-backport_vulnerability_scan copy.yml]: good catch 👍 🙂

**Review by SeriyBg — APPROVED** [review]

File: .github/workflows/tag_image_push.yml [hunk 2]
```diff
@@ -57,18 +67,19 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: hazelcast/hazelcast
-          short-description: Hazelcast IMDG Docker Image
+          repository: ${DOCKER_ORG}/hazelcast
```
Comment (mtyazici) [inline .github/workflows/tag_image_push.yml:70]: I don't think the actions can receive the environment variables in shell style. I believe the line should be as follows
```suggestion
          repository: ${{ env.DOCKER_ORG }}/hazelcast
```

File: .github/workflows/tag_image_push.yml [hunk 3]
```diff
@@ -78,8 +89,8 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: hazelcast/hazelcast-enterprise
-          short-description: Hazelcast IMDG Enterprise Docker Image
+          repository: ${DOCKER_ORG}/hazelcast-enterprise
```
Comment (mtyazici) [inline .github/workflows/tag_image_push.yml:92]: Same as above
```suggestion
          repository: ${{ env.DOCKER_ORG }}/hazelcast-enterprise
```

**Review by mtyazici — CHANGES_REQUESTED** [review]
I left two comments and I will mention another one here, other than that LGTM :+1: 

The file `hazelcast-enterprise/values.yaml` you are deleting here is used by one workflow, instead of deleting it here in another PR we can move it in `.github` folder and update the workflow .

File: .github/workflows/tag_image_push.yml (hunk 2 above)
Comment (hasancelik) [reply .github/workflows/tag_image_push.yml]: Actually, it is working but you are right we should follow convention 👍

**Comment by hasancelik** [comment]
@mtyazici I updated the PR, PTAL

**Review by mtyazici — APPROVED** [review]
:+1:

**Pull request merged**